lib/gamera/page_sections/form.rb

```diff
@@ -93,7 +93,11 @@ def initialize(fields)
       def fill_in_form(fields)
         fields.each do |field, value|
           f = send("#{field}_field")
-          f.set(value) || Array(value).each { |val| f.select(val) }
+          if f.tag_name == "select"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.

[Read more about it here.](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/StringLiterals)